### PR TITLE
MWMVBE2-85: add description field for customer patch operation

### PR DESCRIPTION
**Pull request by KrasnovNikita**

### CODE REVIEW

**Review by sergeysenja1992 — APPROVED** [review]

File: src/main/resources/swagger/api.yml [hunk 1]
```diff
@@ -508,6 +508,9 @@ definitions:
             "path":
                 type: string
                 description: "Json pointer, e.g. '/characteristic/-'"
+            "reason":
+              type: string
+              description: "Operation reason"
```
Comment (vkirichenko) [inline src/main/resources/swagger/api.yml:513]: let's extand model according to TMF recomnadation (for example how it was done on ms communication)

Comment (KrasnovNikita) [reply src/main/resources/swagger/api.yml:513]: @vkirichenko May you give an example on ms communication?

Comment (vkirichenko) [reply src/main/resources/swagger/api.yml:513]: https://github.com/xm-online/tmf-ms-communication/blob/master/src/main/resources/swagger/api-extension.yml

Comment (KrasnovNikita) [reply src/main/resources/swagger/api.yml]: @vkirichenko extension.yml(-s) was added

File: gradle/swagger.gradle [hunk 2]
```diff
@@ -12,33 +12,59 @@ buildscript {
...
+        apiPackage = "com.icthh.xm.tmf.ms.customer.api"
+        modelPackage = "com.icthh.xm.tmf.ms.customer.model"
+        apiFilesConstrainedTo = [""]
+        modelFilesConstrainedTo = [""]
+        supportingFilesConstrainedTo = ["ApiUtil.java"]
+        configOptions = [delegatePattern: "true", title: "customer"]
+        validateSpec = true
+        importMappings = [Problem: "org.zalando.problem.Problem"]
+    }
+    compileJava.dependsOn(taskName)
```
Comment (amedvedchuk) [inline gradle/swagger.gradle:30]: seems like this is a duplication for `createOpenApiGenerateTaskV31`
is it possible to refactor to one method?

Comment (KrasnovNikita) [reply gradle/swagger.gradle]: @amedvedchuk one method was created

**Review by vkirichenko — APPROVED** [review]

**Pull request closed without merging**